package.json

```diff
@@ -0,0 +1,57 @@
+{
```

Comment:
Could you please also add : 
```
    "publishConfig": {
        "access": "public"
    },
```
this will force publishing the package in public (instead of specifying it on the CLI)